quick_setup_script.py

```diff
@@ -257,31 +257,31 @@ def __init__(self, api_host='http://0.0.0.0:9000/api/v1/', email=None, password=
 
 if __name__ == '__main__':
 
-    s = Setup(
-        api_host='https://dev.withsempo.com/api/v1/',
-        api_token=
-        'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1NzcxODU3NzQsImlhdCI6MTU3NjU4MDk3NCwiaWQiOjEsInJvbGVzIjp7IkFETUlOIjoic2VtcG9hZG1pbiJ9fQ.CGrSHCUcpJlGq3sBgim7omwTBZAx-v0N3AAUurlkqhI|eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjI0Mzk2MDQxMjYsImlhdCI6MTU3NTY5MDQ5NiwiaWQiOjJ9.WaSdLvU5aGxLmNo5uZV0_PmV7LOTymeBBxOymy0Er7U'
-    )
-
     # s = Setup(
-    #     api_host='http://0.0.0.0:9000/api/v1/',
+    #     api_host='https://dev.withsempo.com/api/v1/',
     #     api_token=
-    #     'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1NzcxNjc3MDgsImlhdCI6MTU3NjU2MjkwOCwiaWQiOjEsInJvbGVzIjp7IkFETUlOIjoic2VtcG9hZG1pbiJ9fQ.S1dMa668pSLkbNAYzgWOC6Td-E84xgDxEg5ilinKiGI|eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1NzYyMTEzMDIsImlhdCI6MTU3NjEyNDg3MiwiaWQiOjZ9.zkEUtwMgOSrcLy68Rtv_JeMCj9HrsOyQUfH3Dc3itYE'
+    #     'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1NzcxODU3NzQsImlhdCI6MTU3NjU4MDk3NCwiaWQiOjEsInJvbGVzIjp7IkFETUlOIjoic2VtcG9hZG1pbiJ9fQ.CGrSHCUcpJlGq3sBgim7omwTBZAx-v0N3AAUurlkqhI|eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjI0Mzk2MDQxMjYsImlhdCI6MTU3NTY5MDQ5NiwiaWQiOjJ9.WaSdLvU5aGxLmNo5uZV0_PmV7LOTymeBBxOymy0Er7U'
     # )
-    # #
+
+    s = Setup(
+        api_host='http://0.0.0.0:9000/api/v1/',
+        api_token=
+        'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1Nzg2MjE3MzIsImlhdCI6MTU3ODAxNjkzMiwiaWQiOjEsInJvbGVzIjp7IkFETUlOIjoic2VtcG9hZG1pbiJ9fQ.-Luhouipdrtw2mSFTSzlUR3CB5FWPGMLyZP2k6cv2b4|eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1NzYyMTEzMDIsImlhdCI6MTU3NjEyNDg3MiwiaWQiOjZ9.zkEUtwMgOSrcLy68Rtv_JeMCj9HrsOyQUfH3Dc3itYE'
```

Comment:
I feel like these should be in local config or something instead of committing to github? even though it is dev...

Reply:
because they change all the time (every 24 hours) and they're not really great to stick in local config. We could throw them into a second .gitignored file.

Reply:
ok